modules/json/src/parsers/enhanced-fetch.js

```diff
@@ -0,0 +1,34 @@
+import {csvParseRows} from 'd3-dsv';
+
+export default function enhancedFetch(url) {
```

Comment:
@ibgreen we should no longer need this with `@loaders.gl/csv`?

Reply:
Yes... However, I think the CSV loader still only supports batched loading.